keyboards/handwired/hnah40rgb/hnah40rgb.c

```diff
@@ -0,0 +1,46 @@
+#include "hnah40rgb.h"
+#include "config.h"
```

Comment:
This isn't actually needed, and is automatically included in every file. 

```suggestion
```

Reply:
thanks!